pkgs/by-name/un/unfs3/package.nix

```diff
@@ -20,6 +23,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-5iAriIutBhwyZVS7AG2fnkrHOI7pNAKfYv062Cy0WXw=";
   };
 
+  patches = [
+    # Fix implicit declaration warning with GCC 14
+    (fetchpatch {
+      url = "https://gitlab.alpinelinux.org/alpine/aports/-/raw/152dc14a65a89f253294cc5b4c96cf0d6658711a/main/unfs3/implicit.patch";
+      hash = "sha256-YjTVOBpxda5MAYwmtg+I34HgmJDXaTi6xr/XWrOMINg=";
+    })
```

Comment:
Can you use `fetchpatch2` and `?full_index=1`?

Reply:
Changed to `fetchpatch2` (doesn't seem documented/recommended in the manual though? https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches ). `?full_index=1` does not apply here, the patch does not come from GitHub.